samples/snippets/linear_regression_tutorial_test.py

```diff
@@ -52,9 +52,27 @@ def test_linear_regression(random_model_id: str) -> None:
     # Expected output results:
     # index  mean_absolute_error  mean_squared_error  mean_squared_log_error  median_absolute_error  r2_score  explained_variance
     #   0        227.012237         81838.159892            0.00507                173.080816        0.872377    0.872377
-    #   1 rows x columns
+    #   1 rows x 6 columns
     # [END bigquery_dataframes_bqml_linear_evaluate]
+    # [START bigquery_dataframes_bqml_linear_predict]
+    # Select the model you'll use for predictions. `read_gbq_model` loads
+    # model data from BigQuery, but you could also use the `model` object
+    # object from previous steps.
+    model = bpd.read_gbq_model(
+        your_model_id,
+        # For example: "bqml_tutorial.penguins_model",
+    )
+
+    # Use 'contains' function to filter by island containing the string
+    # "Biscoe".
+    biscoe_data = model.loc[model["island"].str.contains("Biscoe")]
+
+    result = model.predict(biscoe_data)
+
+    # Expected output results:
```

Comment:
Could we print out a few rows of the prediction here too. Doesn't have to be many, just enough to get an idea of what the data looks like.